src/components/application_manager/test/commands/mobile/subscribe_button_request_test.cc

```diff
@@ -133,6 +133,31 @@ TEST_F(SubscribeButtonRequestTest, Run_UiIsNotSupported_UNSUCCESS) {
                     .asInt()));
 }
 
+TEST_F(SubscribeButtonRequestTest, Run_UiIsSupported_UNSUCCESS) {
+  MockAppPtr app(CreateMockApp());
+  ON_CALL(mock_app_manager_, application(_)).WillByDefault(Return(app));
+
+  MockHMICapabilities hmi_capabilities;
+  ON_CALL(mock_app_manager_, hmi_capabilities())
+      .WillByDefault(ReturnRef(hmi_capabilities));
+  ON_CALL(hmi_capabilities, is_ui_cooperating()).WillByDefault(Return(true));
+
+  ON_CALL(hmi_capabilities, button_capabilities())
+      .WillByDefault(Return(static_cast<const SmartObject*>(NULL)));
+
+  ON_CALL(*app, IsSubscribedToButton(_)).WillByDefault(Return(true));
```

Comment:
@LevchenkoS I wondered about this, too. I used ON_CALL because this test class has been written by you recently and you've used everywhere ON_CALL, so I thought it is on purpose? Anyway, I think also that is better to check with EXPECT_CALL.